.drone.yml

```diff
@@ -48,6 +89,14 @@ steps:
     tag: "${DRONE_TAG}-amd64"
     username:
       from_secret: docker_username
+    build_args:
+      - "VERSION=${DRONE_TAG}"
+      - "RELEASES=releases.rancher.com"
+    context: package/
+    custom_dns: 1.1.1.1
+  volumes:
```

Comment:
Do you need this docker volume? I think publish step doesn't really need it

Reply:
Ah, good point. I'll remove it

Reply:
@luthermonson do we need this? I believe it's in rancher/dapper

Reply:
Looks like only `dapper ci` needs this volume since it can reuse docker build cache.